test/python/transpiler/test_instruction_durations.py

```diff
@@ -88,3 +60,113 @@ def test_fail_if_get_unbounded_duration_with_unit_conversion_when_dt_is_not_prov
         parameterized_delay = Delay(param, "s")
         with self.assertRaises(TranspilerError):
             InstructionDurations().get(parameterized_delay, 0)
+
+
+class TestInstrctionDurationsFromBackendV1(QiskitTestCase):
+    """Test :meth:`~.from_backend` of :class:`.InstructionDurations` with
+    :class:`.BackendV1`"""
+
+    def setUp(self):
+        super().setUp()
+
+        self.backend = FakeParis()
+        self.backend_config = self.backend.configuration()
+        self.backend_props = self.backend.properties()
+        self.example_qubit = (0,)
+        self.example_gate = "x"
+
+        # Setting dt for the copy of backend to be None
+        self.backend_cpy = deepcopy(self.backend)
+        self.backend_cpy.configuration().dt = None
+
+    def test_backend_dt_equals_inst_dur_dt(self):
+        durations = InstructionDurations.from_backend(self.backend)
+        self.assertEqual(durations.dt, self.backend_config.dt)
+
+    def test_backend_gate_length_equals_inst_dur(self):
+        durations = InstructionDurations.from_backend(self.backend)
+        inst_dur_duration = durations.get(self.example_gate, self.example_qubit[0], "s")
+        backend_inst_dur = self.backend_props.gate_length(
+            gate=self.example_gate, qubits=self.example_qubit
+        )
+        self.assertEqual(inst_dur_duration, backend_inst_dur)
+
+    def test_backend_without_dt_sets_inst_dur_None(self):
+        durations = InstructionDurations.from_backend(self.backend_cpy)
+        self.assertIsNone(durations.dt)
+
+    def test_get_dur_s_with_dt_None(self):
+        durations = InstructionDurations.from_backend(self.backend_cpy)
+        self.assertEqual(
+            durations.get(self.example_gate, self.example_qubit[0], "s"), 3.5555555555555554e-08
+        )
+
+    def test_raise_dur_get_dt_with_backend_dt_None(self):
+        durations = InstructionDurations.from_backend(self.backend_cpy)
+        with self.assertRaises(TranspilerError):
+            durations.get(self.example_gate, self.example_qubit[0])
+
+    def test_works_unequal_dt_dtm(self):
+        self.backend_cpy.configuration().dt = 1.0
+
+        # This is expcted to fail
+        InstructionDurations.from_backend(self.backend_cpy)
+
+        self.backend_cpy.configuration().dt = None  # Resetting to None
```

Comment:
You don't need to reset manually. See the documentation:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp

Reply:
applied suggestion :)